mappings/net/minecraft/client/gui/widget/TextFieldWidget.mapping

```diff
@@ -40,31 +43,53 @@ CLASS net/minecraft/class_342 net/minecraft/client/gui/widget/TextFieldWidget
 	METHOD method_1851 hasBorder ()Z
 	METHOD method_1852 setText (Ljava/lang/String;)V
 		ARG 1 text
+	METHOD method_1853 getDistanceBetween (I)I
+		ARG 1 position1
 	METHOD method_1854 setRenderTextProvider (Ljava/util/function/BiFunction;)V
 		ARG 1 renderTextProvider
 	METHOD method_1855 moveCursor (I)V
+		ARG 1 distance
+	METHOD method_1856 setSelectable (Z)V
+		ARG 1 selectable
 	METHOD method_1858 setHasBorder (Z)V
 		ARG 1 hasBorder
+	METHOD method_1859 getInnerWidth ()I
 	METHOD method_1860 setUneditableColor (I)V
 		ARG 1 color
 	METHOD method_1861 getMaxLength ()I
 	METHOD method_1862 setVisible (Z)V
 		ARG 1 visible
 	METHOD method_1863 setChangedListener (Ljava/util/function/Consumer;)V
+	METHOD method_1864 getDistanceBetween (IIZ)I
+		ARG 1 position1
+		ARG 2 position2
+		ARG 3 stopAtEndOfWord
 	METHOD method_1865 tick ()V
 	METHOD method_1866 getSelectedText ()Ljava/lang/String;
 	METHOD method_1867 addText (Ljava/lang/String;)V
 	METHOD method_1868 setEditableColor (I)V
 		ARG 1 color
+	METHOD method_1869 getDistanceBetween (II)I
+		ARG 1 position1
+		ARG 2 position2
+	METHOD method_1870 moveCursorHome ()V
+	METHOD method_1872 moveCursorToEnd ()V
 	METHOD method_1874 onChanged (Ljava/lang/String;)V
 		ARG 1 newText
 	METHOD method_1875 setCursor (I)V
 		ARG 1 cursor
+	METHOD method_1876 setFocused (Z)V
+		ARG 1 focused
 	METHOD method_1880 setMaxLength (I)V
 		ARG 1 maximumLength
 	METHOD method_1881 getCursor ()I
 	METHOD method_1882 getText ()Ljava/lang/String;
+	METHOD method_1883 setCursorKeepingSelection (I)V
+		ARG 1 cursor
+	METHOD method_1884 setCursorMin (I)V
+		ARG 1 cursorMin
 	METHOD method_1885 isVisible ()Z
+	METHOD method_1886 drawSelectionOverlay (IIII)V
 	METHOD method_1887 setSuggestion (Ljava/lang/String;)V
 		ARG 1 suggestion
 	METHOD method_1888 setIsEditable (Z)V
```

Comment:
I would think the `Is` part here to be a bit redundant.

Reply:
Those mappings were made by Gegy a while ago. I can change them to follow new conventions if breaking changes are OK.